### ISSUE TITLE
Unlimited Item Duping Exploit (arma engine bug -> workaround possible)?

### ISSUE THREAD

**Issue by wiking-at** [A3Wasteland/ArmA3_Wasteland.Altis#386]
There is currently a bug in the engine which allows unlimited duping of items and therefor money when selling (the duped items in the video could all be sold in wasteland).

private video link:
https://www.youtube.com/watch?v=9LvyoP5vfWU
permissions are set to allow access for agentrev's gmail account (the one used for git commits).

I'm not sure if there is any scripting possibility to fix this bullshit.
i try to reach someone from bohemia also for a fix.

**Comment by Staynex** [comment]
FIXED -> https://forums.bistudio.com/topic/190914-arma-3-car-exploit/?p=3091039

**Link History:**
Armajunkies Internal Ticket: https://armajunkies.atlassian.net/projects/AJWL/issues/AJWL-69
BI Forum Report: https://forums.bistudio.com/topic/190914-arma-3-car-exploit/
BI Feedbacktracker Report: https://feedback.bistudio.com/T116866
AgentRev A3Wasteland Report: https://github.com/A3Wasteland/ArmA3_Wasteland.Altis/issues/386
1. Video: https://www.youtube.com/watch?v=9LvyoP5vfWU
2. Video (with Code Examples): https://www.youtube.com/watch?v=ebeQJiLrh4E

**Comment by AgentRev** [comment]
I do not possess the capability to fix this.

**Comment by nerdalertdk** [comment]
not even Bi do :( sad panda

**Comment by codexGW** [comment]
> FIXED -> https://forums.bistudio.com/topic/190914-arma-3-car-exploit/?p=3091039

Wait, so this is fixed right so Agent doesn't need to do anything? I'm confused... but what's new :smile:

**Comment by Staynex** [comment]
@Gigatek1 Yes! Bohemia fixed it! Arma doesn't show the inventory of the dead body if you are near a car anymore :)

**Comment by codexGW** [comment]
Oh, that's a clever fix.
